### DISCUSSION TITLE
only the last usernameSecret in config.yaml is taken

### DISCUSSION THREAD

**Discussion by frankli-n** [splunk/splunk-connect-for-snmp#968] [Q&A] [answered]
I am only getting results for the last usernameSecret given. In the below example only snmpv3-secret-md5-aes returns results.
Does anyone know if this is a bug?

splunk:
  enabled: true
  protocol: http
  host: 192.168.157.134
  token: 9754a1da-f3a6-4582-a3fb-***
  insecureSSL: "true"
  port: "8088"
traps:
  communities:
    2c:
      - public
      - homelab
  loadBalancerIP: 192.168.157.10
  usernameSecrets:
    - snmpv3-secret-md5-aes128
    - snmpv3-secret-sha-aes
    - snmpv3-secret-md5-des
    - snmpv3-secret-md5-aes
  securityEngineId:
    - "80003a8c04" # Testing
    - "80003a8c05" # Testing
    - "80003a8c06" # Testing
    - "80003a8c07" # Testing
  logLevel: "DEBUG"

**Comment by ajasnosz** [comment, marked as answer]
Hello, 
is the username for those usernameSecrets the same? 
If so it's a bug similar to https://github.com/splunk/splunk-connect-for-snmp/issues/930

**Reply by frankli-n** [reply]
Yes its the same issue as that bug. Thank you for directing me to it!
